src/main.rs

```diff
@@ -158,11 +163,8 @@ impl ManifestDelivery {
         match self {
             ManifestDelivery::Repo(_) | ManifestDelivery::Path(_) => {
                 let fq: PathBuf = serve_root.join(unsafe { INDEX_FILE_PATH.as_ref().map(|i| i.to_str().unwrap()).unwrap() });
-                log::data("looking for indexfile at", &fq);
-                {
-                    std::fs::File::open(&fq).map_err(|e| RepoError::IndexFile(e))?;
-                }
-            
+                println!("{:#?}", fq.to_str().unwrap());
```

Comment:
I feel like we lost some useful information here (it's easier to read a log that says "looking for THING {thing}" than "looking for {thing}")

Reply:
I agree, not sure why I did it like this... will revert :+1: